R/provider-databricks.R

```diff
@@ -147,7 +160,12 @@ databricks_workspace <- function() {
 
 # Try various ways to get Databricks credentials. This implements a subset of
 # the "Databricks client unified authentication" model.
-databricks_token <- function(workspace) {
+databricks_token <- function(workspace = databricks_workspace(), token = "") {
+  # Accept any non-empty user-supplied token.
```

Comment:
I'd suggest a `check_string(token, allow_null = TRUE)` here (along with the appropriate `error_call`).

Reply:
I haven't used argument type checking for internal-only functions so far -- should I start?